### PR TITLE
[CELEBORN] Fix push small data

### PR DESCRIPTION
**Pull request by exmy**
## How was this patch tested?
CI

### CODE REVIEW

File: gluten-celeborn/common/src/main/scala/org/apache/spark/shuffle/CelebornPartitionPusher.scala [hunk 1]
```diff
@@ -35,8 +35,8 @@ class CelebornPartitionPusher(
 
   @throws[IOException]
   def pushPartitionData(partitionId: Int, buffer: Array[Byte], length: Int): Int = {
-    logDebug(s"Push record, size ${buffer.length}.")
-    if (buffer.length > clientPushBufferMaxSize) {
+    if (length > clientPushBufferMaxSize) {
```
Comment (ulysses-you) [inline gluten-celeborn/common/src/main/scala/org/apache/spark/shuffle/CelebornPartitionPusher.scala:38]: make sense to me, cc @kerwin-zk

Comment (zhanglistar) [reply gluten-celeborn/common/src/main/scala/org/apache/spark/shuffle/CelebornPartitionPusher.scala:38]: Good catch!

**Comment by kerwin-zk** [comment]
LGTM.Thanks!

**Review by baibaichen — APPROVED** [review]
LGTM

**Pull request merged**